data-layout.md

```diff
@@ -0,0 +1,62 @@
+# Data Layout
+
+An overview of the game vs translation data layout to provide an understanding of how the tool works at a high level.
+
+For specific translation instructions, see [translating.md](translating.md).
+
+First, there are two relevant game directories:
+* `<install dir>`: The .exe file's install directory. You picked this when installing the game. Static game files and our added config files go here.
+* `<local data dir>`: `~/AppData/LocalLow/cygames/umamusume/`. The 'local data directory', where the game downloads new (and old) content.
+
+## Story/Lyrics etc.
+
+### Game
+
+The local data directory contains a file called `meta`, which is an sqlite database mapping story/winning live/etc. Unity files from a standardized filepath (presumably a path on the cygames servers) to a hash of its contents.
```

Comment:
While not wrong, more accurately it's an index of all assets listing all the data the game requires for them.
The path is the internal unity asset path.
Also should be noted the hash is not a simple standard hash of file contents, it's calculated in a custom manner. (details available but not really useful)